### PR TITLE
Added a class to house the application data folder for the project

### PR DESCRIPTION
**Pull request by cjames9001**

### CODE REVIEW

File: WMCDuplicateRemover/Code/Wrappers/MicrosoftEventLogWrapper.cs [hunk 1]
```diff
@@ -9,8 +9,7 @@ namespace WMCDuplicateRemover
     public class MicrosoftEventLogWrapper : EventLogWrapper
     {
         private HashSet<string> eventLogCache;
-        private static string programDataFolder = $"{Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)}\\WMCDuplicateRemover";
-        private readonly string eventLogEntryCacheFilePath = Path.Combine(programDataFolder, "eventLogEntries.cache");
+        private readonly string eventLogEntryCacheFilePath = Path.Combine(StaticValues.WMCDuplicateRemoverApplicationDataFolder, "eventLogEntries.cache");
```
Comment (taylorglenn) [inline WMCDuplicateRemover/Code/Wrappers/MicrosoftEventLogWrapper.cs:12]: Are you just setting up a path for logging the things that happen, or am I missing something here?
Edit: The more I look at it, the more I can see that you are.  Please correct me if I'm wrong.

File: WMCDuplicateRemover/StaticValues.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace WMCDuplicateRemover
+{
+    public static class StaticValues
+    {
+        public static readonly string WMCDuplicateRemoverApplicationDataFolder = $"{Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)}\\WMCDuplicateRemover";
+    }
+}
```
Comment (taylorglenn) [inline WMCDuplicateRemover/StaticValues.cs:9]: Is the point of this class just to house values that don't change?  So you'll just create an instance of this class and use it to reference various values?  (I see that it only has one now, but it seems like you plan to expand it)

File: WMCDuplicateRemoverDriver/Driver.cs [hunk 3]
```diff
@@ -11,8 +11,7 @@ namespace WMCDuplicateRemoverDriver
 {
     public class Driver
     {
-        private static string programDataFolder = $"{Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)}\\WMCDuplicateRemover";
-        private readonly string logPath = Path.Combine(programDataFolder, "WMCDuplicateRemover.log");
+        private readonly string logPath = Path.Combine(StaticValues.WMCDuplicateRemoverApplicationDataFolder, "WMCDuplicateRemover.log");
```
Comment (taylorglenn) [inline WMCDuplicateRemoverDriver/Driver.cs:14]: Do you have two separate logs?  If so, why?

**Comment by taylorglenn** [comment]
Approved.  For the most part, everything makes sense to me.  Creating a class to house static values is an interesting idea, and I hadn't ever thought of it before.

**Comment by cjames9001** [comment]
As we discussed yesterday, there is only the one log, the other file is to store items from the event log. The static class probably not the best place for it but at least it's not duplicated for the time being. I'll go ahead and merge this in.

**Pull request merged**